### PR TITLE
Standings command

### PR DESCRIPTION
**Pull request by skamoen**

### CODE REVIEW

File: f1_schedule_telegram_bot/main.py [hunk 1]
```diff
@@ -107,6 +109,50 @@ def remove_job_if_exists(uid: str, context: ContextTypes.DEFAULT_TYPE) -> bool:
...
+
+    constructor_standing = e.season().get_constructor_standing()
+    driver_standing = e.season().get_driver_standing()
+    races = e.season().get_races()
+
+    table = PrettyTable()
+    table.field_names = ["Position", "Name", "Team", "Points"]
+
+    message = f"Standing after the {races[driver_standing.round_no - 1].race_name} \n"
+    message += "DriversL \n"
```
Comment (martijnjanssen) [inline f1_schedule_telegram_bot/main.py:123]:
```suggestion
    message += "Drivers: \n"
```

Comment (martijnjanssen) [inline f1_schedule_telegram_bot/main.py:123]:
```suggestion
    message += "Drivers: \n"
```

File: f1_schedule_telegram_bot/main.py [hunk 2]
```diff
@@ -251,10 +297,14 @@ def main():
     application = ApplicationBuilder().token(bot_token).build()
 
     start_handler = CommandHandler("start", handle_start)
+    standings_handler = CommandHandler("standings", load_standings)
```
Comment (Fastjur) [inline f1_schedule_telegram_bot/main.py:300]:
```suggestion
    standings_handler = CommandHandler("standings", handle_standings)
```

**Review by Fastjur — APPROVED** [review]
This is a very cool feature, thanks!

**Pull request merged**